frontend/packages/ux-editor/src/components/config/editModal/EditOptions.tsx

```diff
@@ -126,15 +126,14 @@ export function EditOptions<T extends SelectionComponentType>({
         <StudioProperty.Group>
           {component.options?.map((option, index) => {
             const removeItem = () => handleRemoveOption(index);
-            const key = `${option.value}-${index}`; // Figure out a way to remove index from key.
             const optionNumber = index + 1;
             const legend =
               component.type === 'RadioButtons'
                 ? t('ux_editor.radios_option', { optionNumber })
                 : t('ux_editor.checkboxes_option', { optionNumber });
             return (
               <EditOption
-                key={key}
+                key={index}
```

Comment:
Avoid using the index as the key. The reason is that indexes change due to the number of elements in the array.

Providing the index as the key is the same as not providing the key at all. Since React by default uses the index if the key is not provided. Providing the key={index} is just removing the eslint-warning but does not have any effect on the implementation itself. 

Therefore; the previous implementation had a combination between the index and the option to create the key. 

Are there any use-cases where the `option.value` is not unique across options?

Reply:
Great! Please give me a heads-up if you need pair-programming on this one! 💯